### PR TITLE
ci(pr-review): consolidate recurring blind-spot patterns into the three reviewers

### PR DESCRIPTION
**Pull request by trieloff**
## Summary

Folds the recurring-issue catalog proposed in #893 into SLICC's **three existing** automated reviewers (Claude action, Codex, Copilot) instead of standing up a separate webhook reviewer. #893 is closed as superseded by this.

New single source of truth — **`docs/review-patterns.md`** — holds the six blind-spot categories (error-path coverage, UI state preservation, cross-runtime parity, CDP edge cases, native/macOS permissions, test coverage), the five-runtime parity matrix, and the severity rubric. Each reviewer gets an adapter sized to its real limit:

| Reviewer | How it gets the patterns | Limit respected |
|---|---|---|
| **Claude action** | `Read docs/review-patterns.md` added to the workflow prompt (`Read` already allowed) | — |
| **Codex** | condensed `## Automated PR Review Checklist` in root `CLAUDE.md` (`AGENTS.md` symlinks to it) | 30,000-char doc-size budget (now 27,662) |
| **Copilot** | `.github/copilot-instructions.md` + path-scoped `.github/instructions/cross-runtime.instructions.md` | Copilot's 4,000-char truncation (file is 2,440) |

## Precedent corrections

The cited precedents were verified against `gh pr view`; two from #893 were wrong and are fixed here:
- **#565** *added* swift-server signing parity — a positive parity example, not a failure.
- **#453** is port-conflict handling, **not** keychain/entitlements — so the macOS-permissions category no longer cites it (now forward-looking with no bogus citation).
- #566/567 reframed (tool-call cluster collapse-state across reflow); #568 was closed, not merged.

The Augment webhook workflow and its Bedrock/Augment secrets from #893 are **not** adopted.

## Notes

- `npm run lint:docs` passes; files are prettier-formatted.
- Codex/Copilot instruction files only take effect once on `main`; the Claude action's `Read` works as soon as the file lands.
- Per repo convention, please **merge** (no squash).

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .github/instructions/cross-runtime.instructions.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+applyTo: 'packages/{node-server,swift-server}/**'
+---
```
[the remark this answers is not in the document]
Comment (trieloff) [reply .github/instructions/cross-runtime.instructions.md:3]: Good catch — fixed in b0b5eb7f. Switched to the documented comma-separated form `packages/node-server/**,packages/swift-server/**` so the path-specific instructions actually apply (brace expansion isn't supported by Copilot's minimatch-style globbing).
